packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -732,6 +732,9 @@ class _ExpansionTileState extends State<ExpansionTile> with SingleTickerProvider
       case TargetPlatform.windows:
         break;
     }
+    final ShapeBorder? shape = widget.shape ?? expansionTileTheme.shape;
+    final ShapeBorder? collapsedShape = widget.collapsedShape ?? expansionTileTheme.collapsedShape;
+    final ShapeBorder? lerpShape = ShapeBorder.lerp(shape, collapsedShape, _animationController.value);
```

Comment:
I believe the reason for the failing tests is that the animation controller is set up for 0 to represent "collapsed" (and 1 represents "expanded"):

https://github.com/flutter/flutter/blob/0737dbfcb67da7e45c9299143a2d6e5b1a8fdeed/packages/flutter/lib/src/material/expansion_tile.dart#L634-L653

Reply:
We should probably reverse the order here:

```suggestion
    final ShapeBorder? lerpShape = ShapeBorder.lerp(collapsedShape, shape, _animationController.value);
```